database/migrations/2021_08_08_130950_change_upload_multiple_column_in_monsters_table.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class ChangeUploadMultipleColumnInMonstersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('monsters', function (Blueprint $table) {
+            $table->text('upload_multiple')->nullable()->change();
```

Comment:
I would rather change it to longText since we are changing it.

Reply:
I was thinking about it just now 😅
`text` is 65000 chars long, it's enough for some thousand uploads, but we can do it 🤷‍♂️

Reply:
What are thousand uploads in today's age? Peanuts my friend... Peanuts!! :)